fv/containers/containers.go

```diff
@@ -204,6 +222,17 @@ func (c *Container) ExecMayFail(cmd ...string) error {
 	return utils.RunMayFail("docker", arg...)
 }
 
+func (c *Container) ExecOutput(args ...string) (string, error) {
+	arg := []string{"exec", c.Name}
+	arg = append(arg, args...)
+	cmd := exec.Command("docker", arg...)
+	out, err := cmd.Output()
+	if err != nil {
+		return "", err
+	}
```

Comment:
I'd expect it often to be useful to return the output too, when an error occurs.